RationaleMCP/0031/grammar.md

```diff
@@ -173,6 +174,24 @@ end _F;
 > ~~_import-list_ → _IDENT_ ( **,** _IDENT_ )*~~
 
 
+## Clock partitions
+
+> _base-partition_ →\
+> &emsp; **partition** _comment_\
+> &emsp; _clock-clause_ **;**\
+> &emsp; _sub-partition_*
+
+> _sub-partition_ →\
+> &emsp; **subpartition** _comment_\
```

Comment:
Note the presence of _comment_ here, similar to how it was done for _base-partition_.

Reply:
This was brought to attention in today's meeting.  Resolving.